cineast-core/src/main/java/org/vitrivr/cineast/core/data/providers/primitive/NothingProvider.java

```diff
@@ -4,6 +4,8 @@ public class NothingProvider implements PrimitiveTypeProvider {
 
   public static final NothingProvider INSTANCE = new NothingProvider();
 
+  public NothingProvider() {}
```

Comment:
Is there a specific reason you need more instances of that one? We mainly used it as a singleton so far, since all of them are the same anyway.